modules/custom-tuning-specification.adoc

```diff
@@ -68,13 +68,20 @@ The individual items of the list:
     <match> <4>
   priority: <priority> <5>
   profile: <tuned_profile_name> <6>
+  operand: <7>
+    debug: <bool> <8>
+    tunedConfig:
+      reapply_sysctl: <bool> <9>
 ----
 <1> Optional.
 <2> A dictionary of key/value `MachineConfig` labels. The keys must be unique.
 <3> If omitted, profile match is assumed unless a profile with a higher priority matches first or `machineConfigLabels` is set.
 <4> An optional list.
 <5> Profile ordering priority. Lower numbers mean higher priority (`0` is the highest priority).
 <6> A TuneD profile to apply on a match. For example `tuned_profile_1`.
+<7> Optional operand configuration.
+<8> Turn debugging on/off for the TuneD daemon: `true/false` (default is `false`).
```

Comment:
Turn debugging on or off for the TuneD daemon. Options are `true` for on or `false` for off. The default is `false`.